controllers/exercise.controller.js

```diff
@@ -0,0 +1,68 @@
+const db = require('../models/index');
+
+const { Exercise } = db;
+
+const store = async (req, res) => {
+  const exercise = await Exercise.create({
+    title: req.body.title,
+    description: req.body.description,
+    level: req.body.level,
+  });
+
+  return res.json(exercise);
+};
+
+const getAll = async (req, res) => {
+  const exercises = await Exercise.findAll();
+  return res.json(exercises);
+};
+
+const get = async (req, res) => {
+  const exercise = await Exercise.findOne({
+    where: {
+      id: req.params.id,
+    },
+  });
+  if (!exercise) {
+    res.status(404).json({ message: 'Record not found' });
```

Comment:
Add Specified Errors instead of Generic Errors. 
errors.ts
defined all errors with messages and error codes

Reply:
errors.js